### DISCUSSION TITLE
Repeat checkbox

### DISCUSSION THREAD

**Discussion by imodude** [nguyenquyhy/Flight-Tracker-StreamDeck#246] [Q&A]
Hi, I'm trying to setup the GNS big/little left/right knobs. I have a single button press working.
I tried selecting the repeat checkbox, to enable me to hold the button down to continue turning the knobs, but it doesnt seem to work.

Also,...
Is there a gotcha with the stream deck profile editor? I often enter event names or button titles into the custom buttons, but they are occasionally not saved when I go to another button and back again. SO I have to do it again and it often works. Cant work out what I'm doing wrong.

**Comment by rmroc451** [comment]
The gotcha is you have to leave the field you changed in order for it to propagate to your profile. 

Give that a try and let us know :)

**Reply by imodude** [reply]
Thanks - that seems to be the trick. Still cant get the repeat working tho....

**Reply by nguyenquyhy** [reply]
Does it work if you repeatedly tap the button instead of holding? And can you also share you profile?

**Reply by imodude** [reply]
Yes, bashing the button works. I will share my profile... need to work out how to first... :-)

**Reply by imodude** [reply]
Here is my profile. I'm already finding it quite useful for the C172... but still work in progress, and I have to create or copy some icons for the GNS buttons. Just tried again to make sure I wasnt doing something stupid - Checking the repeat box and then going to another button and back again, the repeat buttin is no longer checked.

https://www.dropbox.com/s/ryzsrbotxppjwyw/IMO%20C172.streamDeckProfile?dl=0
